experimental/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ All notable changes to experimental packages in this project will be documented
 * feat(sdk-metrics-base): split up Singular into Sum and Gauge in MetricData [#3079](https://github.com/open-telemetry/opentelemetry-js/pull/3079) @pichlermarc
   * removes `DataPointType.SINGULAR`, and replaces it with `DataPointType.SUM` and `DataPointType.GAUGE`
   * removes `SingularMetricData` and replaces it with `SumMetricData` (including an additional `isMonotonic` flag) and `GaugeMetricData`
+* feat(histogram): align collection of optional Histogram properties with spec [#3102](https://github.com/open-telemetry/opentelemetry-js/pull/3079) @pichlermarc
+  * changes type of `sum` property on `Histogram` to `number | undefined`
+  * changes type of `min` and `max` properties on `Histogram` to `number | undefined`
+  * removes `hasMinMax` flag on `Histogram`
```

Comment:
i understood initally that you removed the flag from the `Histogram` constructor however it's just removed from the export right ?

Reply:
Yes, it is just removed from the export. Added some clarification in [94d384b](https://github.com/open-telemetry/opentelemetry-js/pull/3102/commits/94d384b00a52631e35e122ba2ddefff6cae214e4). :slightly_smiling_face: 

Thanks for bringing this up! :slightly_smiling_face: